linkage/pvs_like_case_study/pvs_like_case_study_sample_data.ipynb

```diff
@@ -15,26 +15,161 @@
    "metadata": {},
```

Comment:
<p>I see that you have a record_id column and a pik column that are always equal (and equal to the index column).  Is that intended?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research_prl/pull/48/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Yes, there is some separate bookkeeping here about records vs PIKs. I've made these columns and not pandas indices because that way they are more obvious and less easy to "lose" accidentally.